kornia/geometry/nerf/data_utils.py

```diff
@@ -0,0 +1,107 @@
+from typing import List, Optional, Tuple, Union
+
+import torch
+from torch.utils.data import BatchSampler, DataLoader, Dataset, RandomSampler, SequentialSampler
+from torchvision.io import read_image
```

Comment:
```suggestion
from kornia.io import load_image
```
no torchvision :)

https://kornia.readthedocs.io/en/latest/io.html

Reply:
OK